### PR TITLE
feat: create TOTP

### PR DESCRIPTION
**Pull request by NicolasLopes7**
This PR implements a new method under the user's client to create TOTP

### CODE REVIEW

File: totp.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package clerk
+
+// TOTP describes a TOTP (Time-based One-Time Password) for a user.
```
Comment (LauraBeatris) [inline totp.go]: Super nit, feel free to ignore. Not sure if starting with the entity is the pattern but it reads a bit repetitive. 

```suggestion
// Describes a Time-based One-time Password (TOTP) for a user.
```

File: user/api.go [hunk 2]
```diff
@@ -23,12 +23,12 @@ func Update(ctx context.Context, id string, params *UpdateParams) (*clerk.User,
...
 }
 
-// UpdateProfileImage sets or replaces the users's profile image.
+// UpdateProfileImage sets or replaces the user's profile image.
 func UpdateProfileImage(ctx context.Context, id string, params *UpdateProfileImageParams) (*clerk.User, error) {
 	return getClient().UpdateProfileImage(ctx, id, params)
 }
 
-// DeleteProfileImage removes the users's profile image.
+// DeleteProfileImage sets or replaces the user's profile image.
```
Comment (LauraBeatris) [inline user/api.go]: I might be missing some context here on that feature, but why a `Delete` method would `set or replace`?

**Review by LauraBeatris — APPROVED** [review]
Looks good! I just left a question, don't want to block the release here.

**Review by mzhong9723 — APPROVED** [review]

File: user/api.go (hunk 2 above)
Comment (NicolasLopes7) [reply user/api.go]: Oh, that was from `go gen ./...`, It generates this api file based on the methods from the other clients, I'll fix those and re-run it! Thanks for spotting that

Comment (NicolasLopes7) [reply user/api.go]: resolved via [5548cc3](https://github.com/clerk/clerk-sdk-go/pull/322/commits/5548cc3627c559f138581078390db4a96d4a319e)

**Pull request merged**